active/community.md

```diff
@@ -0,0 +1,86 @@
+# Online Community Working Group
+
+## Scope of responsibilities
+
+This group exists to support the official community platforms and those in elevated roles on those platforms.
+
+The [Django Forum](https://forum.djangoproject.com/) and the [Django Discord](https://discord.gg/xcRH6mN4fa) are core parts of the overall community and those that help and moderate these platforms take significant time from their day to help others.
+We can support them through training and taking action to make their lives easier.
+
+The broad goals of the group are:
+  - facilitate and centralise discussion between moderators and admins on the various platforms
+  - coordinate efforts for training and processes around moderation
+  - provide support to improve the platforms to lighten moderation and admin workloads
+  - provide training and support to the wider community on how to use the platforms effectively
+  - ensure consistency that our online Django communities are welcoming and safe spaces
+
+- What actions are you proposing the WG be allowed to take directly?
+  - Creation and management of processes related to community platforms.
+    - Adding/removing moderators & other roles
+    - Training for moderators
+    - Ensuring moderation is consistent across teams and platforms
+  - Taking ideas from the community to improve the community spaces.
+- Which actions will the WG take back to the Board for votes?
+  - Moving platforms if required
+  - Budget to improve existing platforms
+
+There is a lot of potential ideas in this space. Part of the WG will be setting up a process to take ideas and have a backlog of these ideas.
+
+### Delegated responsibilities
+
+With regards to Django Software Foundation responsibilities and resources, the group operates with:
+
+- Admin privileges to members of the forum & Discord or existing admin representive joining as a member.
+
+## Initial membership
+
+- Chair: TBD
+- Co-Chair: TBD
+- Board Liaison (must be an active Board member; may be the same as Chair/Co-Chair): TBD
+- Other members:
+  - Andrew Miller
+
+## Future membership
+
+Answer the following questions here:
+
+- Who is eligible to join? Any volunteer, or are there specific requirements?
+  - Anyone can join, but they at least need to be known within the community.
```

Comment:
This sounds too fuzzy to me. I’d recommend replacing with a more specific "active on one of the group’s target platforms", or "a DSF Individual Member", or both of those things.

Reply:
I dropped the DSF Individual member requirement on Jacob's recommendation since being a member of a WG is an option to become an Individual member. 

That said I agree the wording can be improved.